### ISSUE TITLE
Feature Request: Funds

### ISSUE THREAD

**Issue by hugbubby** [manifoldmarkets/manifold#537]
In real financial markets, if you or your firm has a strong track record, or some similar legible argument for your ability to beat other traders, you can ask others to trust you with their money in standardized deals. This causes some Mowshowitzian traps, but on net I'd say it's good; it gives people the opportunity to delegate others who might be better equipped or more intelligent to make financial decisions in the market.

Similarly, I think it would be a fun idea for users of Manifold to be able to create and advertise "funds" they could use to exploit leverage opportunities. Giving high-performing users the ability to trade using "funds" this way seems like it would solve many aspects of the liquidity problems pointed out by Scott, without inflating manifold's economy or requiring a complicated automatic margin call system. In addition, the potential new community dynamic, where the performance of "superforecasters" becomes a topic of discussion, and some of them get to give mock speeches to their investors about "[failing to keep the ship afloat](https://www.youtube.com/watch?v=LI395YShGRQ)" after bad calls, also seems like it would be incredibly entertaining and drive community engagement.

**Comment by akrolsmir** [comment]
Yup! This is in my head one of the first "fun" uses for Manifund (http://mni.fo/manifund), letting people set up an investment vehicle in their trading performance.

Another interesting twist is to set one of just for market creation fees, or just for tips - these each represent different things that are interesting to place a bet on!
